File: lista de exercicios/questão 10/questao.go
package main

import "fmt"

// Função para somar dois números
func SomarDois(a, b int) int {
    return a + b
}

// Função para somar três números
func SomarTres(a, b, c int) int {
    return a + b + c
}

func main() {
    fmt.Println(SomarDois(2, 3))    // Soma de dois números
    fmt.Println(SomarTres(2, 3, 4)) // Soma de três números
}

File: lista de exercicios/questão 10/questao.py
class Calculadora:
    def somar(self, a, b, c=0):
        return a + b + c

# Exemplo de uso:
calc = Calculadora()
print(calc.somar(2, 3))        # Soma de dois números
print(calc.somar(2, 3, 4))     # Soma de três números
